package.json

```diff
@@ -35,6 +35,7 @@
   "scripts": {
     "build:clean": "rm -rf dist",
     "build": "pnpm build:clean && tsup",
+    "publish": "pnpm run build && npm publish",
```

Comment:
Probably add a dry run publish command too.  Good to use these so we never use npm, only pnpm